### PR TITLE
Add new publisher TechCrunch.com

### PR DESCRIPTION
**Pull request by jannichorst**

### CODE REVIEW

File: src/fundus/publishers/us/__init__.py [hunk 1]
```diff
@@ -40,6 +41,20 @@ class US(PublisherEnum):
         parser=CNBCParser,
     )
 
+    TechCrunch = PublisherSpec(
+        name="TechCrunch",
+        domain="https://www.techcrunch.com/",
```
Comment (MaxDall) [inline src/fundus/publishers/us/__init__.py:46]:
```suggestion
        domain="https://techcrunch.com/",
```

File: src/fundus/publishers/us/techcrunch.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime.datetime]:
+            # return generic_date_parsing(self.precomputed.ld.get_value_by_key_path(["NewsArticle", "datePublished"]))
+            return generic_date_parsing(self.precomputed.ld.bf_search("datePublished"))
+
+        @attribute
+        def title(self) -> Optional[str]:
+            # title: Optional[str] = self.precomputed.ld.get_value_by_key_path(["NewsArticle", "headline"])
+            # return title
```
Comment (MaxDall) [inline src/fundus/publishers/us/techcrunch.py:44]:
```suggestion
```

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding 👍 this one looks really good! Just two minor things.

**Comment by jannichorst** [comment]
Glad it looks good! Committed the changes. Let me know if anything else has to be done :)

**Review by MaxDall — DISMISSED** [review]

**Pull request merged**